rules/group.go

```diff
@@ -90,6 +91,7 @@ type GroupOptions struct {
 	Rules             []Rule
 	ShouldRestore     bool
 	Opts              *ManagerOptions
+	RuleQueryOffset   *time.Duration
```

Comment:
All public methods/attributes get the `Rule` prefix.

Reply:
I would think that since this field is always accessed via its `GroupOptions` struct context, just `QueryOffset` would suffice?